pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -154,6 +157,10 @@ func (i *CommandInitOption) Validate(parentCommand string) error {
 		return fmt.Errorf("when etcd storage mode is PVC, storageClassesName is not empty. See '%s init --help'", parentCommand)
 	}
 
+	if i.WaitComponentReadyTimeout < 0 {
+		return fmt.Errorf("wait-component-ready-timeout must be greater than or equal to 0")
```

Comment:
Can the timeout equal `0`?

Reply:
0 is a viable option, representing never exit

Reply:
Good! If `0` means wait for ever, then we'd better explain it in flag usage, because it might be useful.